### PR TITLE
[Docs] Fix docstring format/publish

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: mlrun/db/httpdb.py [hunk 1]
```diff
@@ -3989,10 +3996,11 @@ def store_alert_config(
     ) -> AlertConfig:
         """
         Create/modify an alert.
+
         :param alert_name: The name of the alert.
         :param alert_data: The data of the alert.
-        :param project: the project that the alert belongs to.
-        :return: The created/modified alert.
+        :param project:    The project that the alert belongs to.
+        :return:           The created/modified alert.
```
Comment (TomerShor) [inline mlrun/db/httpdb.py]: I see the `param`s and `return` sometimes has a blank line between them and sometimes not.
What is the correct format and is it displayed differently?

File: mlrun/db/httpdb.py [hunk 2]
```diff
@@ -4012,9 +4020,11 @@ def store_alert_config(
     def get_alert_config(self, alert_name: str, project="") -> AlertConfig:
         """
         Retrieve an alert.
+
         :param alert_name: The name of the alert to retrieve.
-        :param project: The project that the alert belongs to.
-        :return: The alert object.
+        :param project:    The project that the alert belongs to.
+
+        :returns:           The alert object.
```
Comment (TomerShor) [inline mlrun/db/httpdb.py:4030]: Sometime you use `return` and sometimes `returns`. What is the correct format and are both options displayed differently?

File: mlrun/db/httpdb.py (hunk 1 above)
Comment (jillnogold) [reply mlrun/db/httpdb.py]: The blank line (or not) does not affect the display.

File: mlrun/db/httpdb.py (hunk 2 above)
Comment (jillnogold) [reply mlrun/db/httpdb.py:4030]: Updated all to [returns](url)

**Review by TomerShor — APPROVED** [review]

**Review by TomerShor — APPROVED** [review]

**Pull request merged**